ddl/ddl_api.go

```diff
@@ -7033,6 +7032,14 @@ func (d *ddl) createIndex(ctx sessionctx.Context, ti ast.Ident, keyType ast.Inde
 		return d.addHypoIndexIntoCtx(ctx, ti.Schema, ti.Name, indexInfo)
 	}
 
+	// We need to set `IndexPartSpecification.Expr` because json cannot unmarshal it correctly.
+	indexPartSpecificationsArg := make([]*ast.IndexPartSpecification, len(indexPartSpecifications))
```

Comment:
Is this change related to this PR?

Reply:
This PR will create a system table with multi-value index. If this change not included, some tests will fail

Reply:
Got it.